addons/slideshow/functions/fnc_autoTransition.sqf

```diff
@@ -34,6 +35,9 @@ missionNamespace setVariable [_varString, _currentSlide];
     _x setObjectTextureGlobal [0, _images select _currentSlide];
 } count _objects;
 
+//Raise event upon slide change
+[QGVAR(slideChanged), [_images select _currentSlide, _currentSlideshow]] call CBA_fnc_localEvent;
```

Comment:
`_images select _currentSlide` save this into a variable on L33 and use it here and in a loop.

Reply:
Added the variable add swapped all occurrences of  `_images select _currentSlide `95fb60a356789c4e179afdbbad07c19d94eac9d3